sources/Application/Instruments/WavFile.cpp

```diff
@@ -122,13 +122,14 @@ WavFile *WavFile::Open(const char *path) {
 	position+=wav->readBlock(position,4) ;
 	memcpy(&chunk,wav->readBuffer_,4) ;
 	chunk = Swap32(chunk);
-		
+
+    #ifdef PARANOIDWAVFMTVALIDATION
```

Comment:
So we're simply skipping these validations or is this define not in this commit?